src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -12,7 +12,7 @@ export default gql`
         levelNumber
         reviewerId
         isLastLevel
-        templateSectionRestrictions
+        allowedSections
```

Comment:
Do we also want to get `canSelfAssign` and `restrictions` here as well?

Reply:
The only field present in `reviewAssignment` is `allowedSections`. The other mentioned fields are part of `templatePermission` and used for generating the next assignments based on `restrictions` and if the reviewer to be assigned `canSelfAssign`.